File: openapi/code_samples/csharp/api_company_{cui}_risk_compute.cs
using Microsoft.AspNetCore.Mvc;
using Swashbuckle.AspNetCore.Annotations;
using System.Collections.Generic;

[ApiController]
[Route("api/[controller]")]
public class CompanyController : ControllerBase
{
    [HttpGet]
    [Route("risk/{cui}")]
    [SwaggerOperation(Summary = "Compute the risk of a given company CUI.")]
    [ProducesResponseType(typeof(List<RiskScore>), 200)]
    [ProducesResponseType(typeof(Fault), 400)]
    public IActionResult GetCompanyRisk([FromRoute] string cui, [FromQuery(Name = "currency")] string currency = "RON")
    {
        // Your implementation here to compute the risk
        List<RiskScore> riskScores = new List<RiskScore>
        {
            new RiskScore { Score = 75, Category = "Moderate" },
            new RiskScore { Score = 60, Category = "Low" }
        };

        return Ok(riskScores);
    }
}

public class RiskScore
{
    public int Score { get; set; }
    public string Category { get; set; }
}

File: openapi/code_samples/python/api_company_{cui}_risk_compute.py
from flask import Flask, request, jsonify

app = Flask(__name__)

@app.route('/api/company/risk/<string:cui>', methods=['GET'])
def get_company_risk(cui):
    currency = request.args.get('currency', 'RON')
    # Your implementation here to compute the risk
    risk_scores = [
        {'Score': 75, 'Category': 'Moderate'},
        {'Score': 60, 'Category': 'Low'}
    ]
    return jsonify(risk_scores), 200

if __name__ == '__main__':
    app.run()
